OSIsoft_Demo/src/OSIsoftServer.java

```diff
@@ -0,0 +1,56 @@
+import com.ewon.ewonitf.EWException;
+import com.ewon.ewonitf.ScheduledActionManager;
+
+/**
+ * OSIsoftServer class

```

Comment:
Thoughts on not re-using the class name in the comments here or below? Makes it easier to rename the class in the future.

Reply:
Removed this